### PR TITLE
fix: Upgraded client to 1.0.0-beta.1 and demonstrated preview setup

### PR DESCRIPTION
**Pull request by JigarJoshi**
- Updated the Tigris Java SDK to 1.0.0-beta.1.
- Demonstrated the preview setup using `tigris_cloud` spring profile.

### CODE REVIEW

File: src/main/java/com/tigrisdata/starter/spring/TigrisSpringConfiguration.java [hunk 1]
```diff
@@ -32,15 +39,25 @@ public TigrisDatabase tigrisDatabase(
...
+      @Value("${tigris.auth.appSecret:#{null}}") Optional<String> applicationSecret) {
+
+    log.info("Initializing tigris client with server.url={}", serverURL);
+
+    TigrisConfiguration.Builder configBuilder = TigrisConfiguration.newBuilder(serverURL);
+
     if (usePlainText) {
-      networkConfigBuilder.usePlainText();
+      configBuilder.withNetwork(
+          TigrisConfiguration.NetworkConfig.newBuilder().usePlainText().build());
```
Comment (ovaistariq) [inline src/main/java/com/tigrisdata/starter/spring/TigrisSpringConfiguration.java:52]: What does usePlainText() mean ?

File: src/main/java/com/tigrisdata/starter/collections/User.java [hunk 2]
```diff
@@ -15,11 +15,11 @@
 
 import com.tigrisdata.db.annotation.TigrisField;
 import com.tigrisdata.db.annotation.TigrisPrimaryKey;
-import com.tigrisdata.db.type.TigrisCollectionType;
+import com.tigrisdata.db.type.TigrisDocumentCollectionType;
```
Comment (ovaistariq) [inline src/main/java/com/tigrisdata/starter/collections/User.java:18]: We need to make similar changes to quickstart java code as well

File: src/main/resources/application-tigris_preview.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+server:
```
Comment (ovaistariq) [inline src/main/resources/application-tigris_preview.yml:1]: Rename the file to application-tigris-cloud.yaml

File: src/main/resources/application-tigris_preview.yml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  port: 8080
+
+
+## Tigris preview configuration
```
Comment (ovaistariq) [inline src/main/resources/application-tigris_preview.yml:5]: Tigris cloud config

File: src/main/resources/application-tigris_preview.yml (hunk 3 above)
Comment (JigarJoshi) [reply src/main/resources/application-tigris_preview.yml:1]: I will rename it to `application-tigris_cloud`.

File: src/main/java/com/tigrisdata/starter/collections/User.java (hunk 2 above)
Comment (JigarJoshi) [reply src/main/java/com/tigrisdata/starter/collections/User.java:18]: Yes I am working on that.

File: src/main/java/com/tigrisdata/starter/spring/TigrisSpringConfiguration.java (hunk 1 above)
Comment (JigarJoshi) [reply src/main/java/com/tigrisdata/starter/spring/TigrisSpringConfiguration.java:52]: no TLS.

**Review by ovaistariq — APPROVED** [review]

**Pull request merged**